pom.xml

```diff
@@ -2192,6 +2192,16 @@ flexible messaging model and an intuitive client API.</description>
           <artifactId>build-helper-maven-plugin</artifactId>
           <version>${build-helper-maven-plugin.version}</version>
         </plugin>
+        <plugin>
+          <groupId>org.owasp</groupId>
+          <artifactId>dependency-check-maven</artifactId>
+          <version>${dependency-check-maven.version}</version>
+          <configuration>
+            <nvdApiKeyEnvironmentVariable>NIST_NVD_API_KEY</nvdApiKeyEnvironmentVariable>
+            <!-- Uncomment the following to use the NVD data feed provided by the Dependency-Check project -->
+            <!-- <nvdDatafeedUrl>https://jeremylong.github.io/DependencyCheck/hb_nvd/</nvdDatafeedUrl> -->
```

Comment:
I went to this url and am getting a 404 error. Not sure what's up with that.

Otherwise LGTM

Reply:
There's no index for the url, but it does work for the dependency check plugin. The files served by the url are https://github.com/jeremylong/DependencyCheck/tree/gh-pages/hb_nvd . This isn't an official cache so that's why I didn't enable it by default.